localstack/constants.py

```diff
@@ -5,12 +5,6 @@
 # LocalStack version
 VERSION = localstack.__version__
 
-# constant to represent the "local" region, i.e., local machine
-REGION_LOCAL = "local"
-
```

Comment:
'REGION_LOCAL' logic was removed in https://github.com/localstack/localstack/pull/9750